src/worker.py

```
@@ -808,21 +971,36 @@ async def api_register(req, env):
         return err("Registration failed — please try again", 500)
 
-    token = create_token(uid, username, role, env.JWT_SECRET)
-
     await _seed_notification_preferences(env, uid)
 
+    # Generate email verification token (single-use, 24-hour expiry)
+    v_token    = generate_secure_token()
+    v_hash     = hash_token(v_token)
+    v_id       = new_id()
+    expires_at = (
+        datetime.datetime.utcnow() + datetime.timedelta(hours=24)
```

Ruff (0.15.15)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)